### ISSUE TITLE
Add Hub of Hope to Signposting

### ISSUE THREAD

**Issue by emcglackenr1** [CodeYourFuture/CYF-Staff-Tickets#1017]
**Bacgkground** 
Resource recommended in [suicide prevention training](https://www.zerosuicidealliance.com/suicide-awareness-training) was [Hub of Hope](https://hubofhope.co.uk/) Add to signposting tool. This is a Uk only resource. 

Completion criteria 
- Add to signposting website resources for UK regions
- [ ] GLA
- [ ] WM
- [ ] NW
- [ ] LDN

Potential action:
- [ ] Sheff ? - Dependent on #1021 
- potentially move to another ticket
